src/FunctionWrappersWrappers.jl

```diff
@@ -4,20 +4,22 @@ using FunctionWrappers
 
 export FunctionWrappersWrapper
 
-struct FunctionWrappersWrapper{FW}
+struct FunctionWrappersWrapper{FW,FB}
   fw::FW
+  original::Any
```

Comment:
can you extract `original` from `fw`?
E.g., `first(fww.fw).obj[]`?

Reply:
you can. I wasn't sure if you'd prefer that.

Reply:
Probably better not to store the function at more places/avoid adding a field that is unused when `FB===false`.